org-eclipse-che-jdt-ls-extension/jdt.ls.extension.core/src/main/java/org/eclipse/che/jdt/ls/extension/core/internal/navigation/FindImplementersHandler.java

```diff
@@ -119,19 +123,21 @@ private static void findTypesWithSubMethods(
       if (method == null) {
         continue;
       }
-      SymbolInformation openDeclaration = convertToSymbolInformation(method);
-      implementers.add(openDeclaration);
+      Location location = JavaModelUtil.toLocation(method);
+      if (location != null) {
+        SymbolInformation openDeclaration = convertToSymbolInformation(method, location);
+        implementers.add(openDeclaration);
+      }
     }
   }
 
   @SuppressWarnings("restriction")
-  private static SymbolInformation convertToSymbolInformation(IJavaElement javaElement)
-      throws JavaModelException {
+  private static SymbolInformation convertToSymbolInformation(
```

Comment:
If I understand it correctly, both call sites us the same method to compute the location. Why not just move it inside the "convertToSymbolInformation" method?

Reply:
updated